### PR TITLE
Add rancher CLI 2.4.11

### PR DESCRIPTION
**Pull request by abryko**

### CODE REVIEW

File: pkgs/rancher-cli.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  src = fetchFromGitHub {
+    owner = source.owner;
+    repo = source.repo;
+    rev = "v${version}";
+    sha256 = source.sha256;
+  };
+
+  vendorSha256 = source.vendorSha256;
+
+  doCheck = false;
```
Comment (eonpatapon) [inline pkgs/rancher-cli.nix:20]: Tests are failing ?

File: pkgs/rancher-cli.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    '';
+
+  CGO_ENABLED = 0;
+
+  ldflags = ''
+      -X github.com/vmware-tanzu/velero/pkg/buildinfo.Version=${source.version}
+      -X github.com/vmware-tanzu/velero/pkg/buildinfo.GitSHA=${source.rev}
+      -X main.VERSION=${source.version}
+      -w -s -extldflags -static
+  '';
```
Comment (eonpatapon) [inline pkgs/rancher-cli.nix:33]: This should be a list: https://nixos.org/manual/nixpkgs/stable/#ssec-go-common-attributes

File: pkgs/rancher-cli.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  postInstall = ''
+      mv $out/bin/cli $out/bin/rancher
+    '';
+
+  CGO_ENABLED = 0;
+
+  ldflags = ''
+      -X github.com/vmware-tanzu/velero/pkg/buildinfo.Version=${source.version}
+      -X github.com/vmware-tanzu/velero/pkg/buildinfo.GitSHA=${source.rev}
```
Comment (eonpatapon) [inline pkgs/rancher-cli.nix:30]: version and rev are not related to velero is this necessary ?

File: pkgs/rancher-cli.nix (hunk 1 above)
Comment (abryko) [reply pkgs/rancher-cli.nix]: nope, C/C too fast, I'm re-enabling then

**Review by eonpatapon — APPROVED** [review]

**Pull request merged**